examples/prefetch/pages/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Link from 'next/link'
+import fetch from '../libs/fetch'
+
+import useSWR, { mutate } from 'swr'
+
+export default () => {
+  const { data } = useSWR('/api/data', fetch)
+
+  React.useEffect(() => {
+    if (!data) return
+    if (data.length === 0) return
+    data.forEach(project => {
+      fetch(`https://api.github.com/repos/${project}`)
+        .then(projectData => {
+          mutate(`/api/data?id=${project}`, projectData, false)
+        })
+    })
+  }, [data]);
+
+  return <div style={{ textAlign: 'center' }}>
+    <h1>Trending Projects</h1>
```

Comment:
Can you also add a preload tag here:
```js
<NextHead>
  <link preload="/api/data" as="fetch" crossOrigin="anonymous" />
</NextHead>
```

So it will preload even before JS starts to download.  I think these two (preload | prefetch) are good examples and worth mentioning in the readme and website.

Reply:
I was thinking of using a second example for the preload one, would you prefer to have them in the same example?

Reply:
Yeah, kinda because if you want to do some optimization, it's great to do both. But also we can add some other cool examples e.g.: 
- wrap the prefetch part as a function and call it when hovering the link
- call the prefetch outside of the component (so it starts to load before React does)

Reply:
I added all of this, and mentioned in the READMe and you may not need all of them at the same time